spec/features/viewing_peep_spec.rb

```diff
@@ -0,0 +1,20 @@
+feature 'Viewing Peeps' do
+  before(:each) do
+    user = create_test_user
+    create_test_peep(peep: 'first peep', user: user, time: Time.gm(2016, 1, 3, 12, 10, 10))
```

Comment:
Metrics/LineLength: Line is too long. [91/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)